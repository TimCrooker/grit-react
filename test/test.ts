import path from 'path'
import { Grit } from 'grit-cli'

const generator = path.join(__dirname, '..')

test('defaults', async () => {
	const grit = new Grit({
		generator,
		mock: true
	})
  await grit.run()

  expect(await grit.getOutputFiles()).toMatchSnapshot()
})
