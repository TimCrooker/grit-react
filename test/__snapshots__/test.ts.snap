// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`defaults 1`] = `
Array [
  ".gitignore",
  "LICENSE",
  "README.md",
]
`;
